pulp_container/tests/functional/api/test_rbac_remotes.py

```diff
@@ -0,0 +1,139 @@
+# coding=utf-8
+"""Tests that container remotes have RBAC."""
+from random import choice
+import unittest
+
+from pulp_smash import utils
+from pulp_smash.pulp3.bindings import monitor_task
+from pulp_smash.pulp3.constants import ON_DEMAND_DOWNLOAD_POLICIES
+
+from pulp_container.tests.functional.utils import (
+    del_user,
+    gen_container_remote,
+    gen_user,
+    skip_if,
+)
+
+from pulpcore.client.pulp_container.exceptions import ApiException
+
+
+class RBACRemotesTestCase(unittest.TestCase):
+    """RBAC remotes."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Create class-wide variables and prepare api users."""
+        cls.user1 = gen_user(["container.add_containerremote"])
+        cls.user2 = gen_user(["container.view_containerremote"])
+        cls.user3 = gen_user([])
+        cls.remote = None
+
+    @classmethod
+    def tearDownClass(cls):
+        """Delete api users."""
+        del_user(cls.user1)
+        del_user(cls.user2)
+        del_user(cls.user3)
+
+    def test_01_create_remote(self):
+        """Create a remote."""
+        body = _gen_verbose_remote()
+        type(self).remote = self.user1["remote_api"].create(body)
+        for key in ("username", "password"):
+            del body[key]
+        for key, val in body.items():
+            with self.subTest(key=key):
+                self.assertEqual(self.remote.to_dict()[key], val, key)
+
+    @skip_if(bool, "remote", False)
+    def test_02_create_same_name(self):
```

Comment:
I am not entirely sure I see benefit in this exact test what would focus on the rbac testing itself, can you elaborate?

Reply:
I must confess, i just took the tests from crud_remotes and added some rbac.
+1 to be more specific about what we test.